src/components/Button.tsx

```diff
@@ -55,6 +65,7 @@ export default React.memo(function Button(props: ButtonProps) {
     loadingColor,
     touchableStyle,
     iconMargin = 4,
+    fontStyle = styles.fontStyle,
```

Comment:
Confirming with design on this , but all buttons in the earn flow require buttons with the small font size instead of medium. 
We could also make a TextSizes enum similar to BtnSizes, so that way consumers only have limited ways of tweaking the style. Thoughts?

Reply:
I think yes to TextSizes enum. These Button components are easy to work with. lets keep it that way